File: java/src/main/java/de/gdata/vaas/messages/VaasOptions.java
package de.gdata.vaas.messages;

import com.google.gson.annotations.SerializedName;
import lombok.Getter;
import lombok.Setter;

public class VaasOptions {

    @Getter
    @Setter
    @SerializedName("use_hash_lookup")
    boolean UseHashLookup;

    @Getter
    @Setter
    @SerializedName("use_cache")
    boolean UseCache;

    public VaasOptions() {
        this.UseCache = true;
        this.UseHashLookup = true;
    }
}

File: php/src/vaas/VaasOptions.php
<?php

namespace VaasSdk;

class VaasOptions
{
    public bool $UseCache = true;
    public bool $UseHashLookup = true;

    public function __construct(bool $useCache = true, bool $useHashLookup = true)
    {
        $this->UseCache = $useCache;
        $this->UseHashLookup = $useHashLookup;
    }
}